web/packages/teleterm/src/ui/Vnet/VnetConnectionItem.tsx

```diff
@@ -69,168 +69,167 @@ export const VnetSliderStepHeader = (props: { goBack: () => void }) => (
   />
 );
 
-const VnetConnectionItemBase = forwardRef(
-  (
-    props: {
-      onClick: () => void;
-      title: string;
-      showBackButton?: boolean;
-      showHelpButton?: boolean;
-      isActive?: boolean;
-      tabIndex?: number;
-    },
-    ref
-  ) => {
-    const { status, start, stop, startAttempt, stopAttempt } = useVnetContext();
-    const isProcessing =
-      startAttempt.status === 'processing' ||
-      stopAttempt.status === 'processing';
-    const indicatorStatus =
-      startAttempt.status === 'error' || stopAttempt.status === 'error'
-        ? 'error'
-        : status === 'running'
-          ? 'on'
-          : 'off';
+interface VnetConnectionItemBaseProps {
```

Comment:
What's the reason for extracting the interface?

Reply:
Because I thought I need to repeat it twice in the definition below. It turns out it's enough to do it once, in the `forwardRef` template parameters.